front/lib/auth.ts

```diff
@@ -894,3 +895,14 @@ export async function prodAPICredentialsForOwner(
     workspaceId: owner.sId,
   };
 }
+
+export function getGroupIdsFromHeaders(
```

Comment:
Not sure I'd put this here it's okay to have a kind of request helper so we can also use the type from NextJS.